spec/request/v1/facilities/ccp_request_spec.rb

```diff
@@ -10,496 +10,527 @@
+              bbox: ['-112.54', '32.53', '-111.04', '34.03'],
+              type: 'provider',
+              specialties: ['213E00000X']
+            }
+          end
+
+          context 'specialties=261QU0200X' do
+            let(:params) do
+              {
+                address: 'South Gilbert Road, Chandler, Arizona 85286, United States',
+                bbox: ['-112.54', '32.53', '-111.04', '34.03'],
+                type: 'provider',
+                specialties: ['261QU0200X']
+              }
+            end
+
+            it 'returns a results from the pos_locator' do
+              get '/v1/facilities/ccp', params: params
+
+              bod = JSON.parse(response.body)
+
+              sha256 = if Flipper.enabled?(:facility_locator_ppms_use_v1_client)
+                         '104ecb200e22dfd96f759bcdd27c6c174ee35c02394bc4d199aaccb51d8486f3'
+                       else
+                         '398681135712746c43545dad381cacaba234e249f02459246ae709a6200f6c41'
+                       end
+
+              expect(bod['data']).to include(
+                {
+                  'id' => sha256,
+                  'type' => 'provider',
+                  'attributes' => {
+                    'acc_new_patients' => 'true',
+                    'address' => {
+                      'street' => '3200 S Gilbert Rd',
+                      'city' => 'Chandler',
+                      'state' => 'AZ',
+                      'zip' => '85286'
+                    },
+                    'caresite_phone' => '4808275700',
+                    'email' => nil,
+                    'fax' => nil,
+                    'gender' => 'NotSpecified',
+                    'lat' => 33.259952,
+                    'long' => -111.790163,
+                    'name' => 'Banner Urgent Care Services LLC',
+                    'phone' => nil,
+                    'pos_codes' => '20',
+                    'pref_contact' => nil,
+                    'unique_id' => '1609229764'
+                  },
+                  'relationships' => {
+                    'specialties' => {
+                      'data' => []
+                    }
+                  }
+                }
+              )
+              expect(response).to be_successful
+            end
+          end
+
+          it "sends a 'facilities.ppms.request.faraday' notification to any subscribers listening" do
+            allow(StatsD).to receive(:measure)
+
+            expect(StatsD).to receive(:measure).with(
+              'facilities.ppms.provider_locator',
+              kind_of(Numeric),
+              hash_including(
+                tags: ['facilities.ppms']
+              )
+            )
+
+            expect(StatsD).to receive(:measure).with(
+              'facilities.ppms.providers',
+              kind_of(Numeric),
+              hash_including(
+                tags: ['facilities.ppms']
+              )
+            ).exactly(7).times
```

Comment:
Why did this number change?

Reply:
This is a bugfix, we weren't deduplicating results correctly